climada/util/test/test_coordinates.py

```diff
@@ -864,7 +864,7 @@ def test_get_admin1_info_pass(self):
         self.assertEqual(len(admin1_info['CHE']), 26)
         self.assertEqual(len(admin1_shapes['IDN']), 33)
         self.assertEqual(len(admin1_info['USA']), 51)
-        self.assertEqual(admin1_info['USA'][1][4], 'US-WA')
+        self.assertEqual(admin1_info['USA'][1]['iso_3166_2'], 'US-ID')
```

Comment:
Note that this won't work when running the unit tests with the old Natural Earth versions because the ordering of US states has changed.